### PR TITLE
Get rid of SSRF

### PR DESCRIPTION
**Pull request by anthony07**
Error messages with the remote download is pretty open and I think it's a big vulnerability issue.

### CODE REVIEW

File: lib/carrierwave/uploader/download.rb [hunk 1]
```diff
@@ -43,7 +43,8 @@ def file
           @file
 
         rescue StandardError => e
-          raise CarrierWave::DownloadError, "could not download file: #{e.message}"
+          reason = e.is_a?(Net::HTTPBadResponse) || e.is_a?(Errno::EHOSTUNREACH) ? '' : ": #{e.message}"
```
Comment (bensie) [inline lib/carrierwave/uploader/download.rb]: Can you explain this further? Why are messages swallowed for these two exception classes?

**Comment by thomasfedb** [comment]
@anthony07 Could you provide some more context on the situations in which this change is helpful? Some examples of what was wrong the the old behaviour would be good.

File: lib/carrierwave/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module CarrierWave
-  VERSION = "0.10.0"
+  VERSION = "0.10.2"
```
Comment (thomasfedb) [inline lib/carrierwave/version.rb]: We can't merge a PR that changes `version.rb`.

**Comment by thomasfedb** [comment]
@anthony07 I'm closing this PR as I've not heard back from you. Let me know if you'd like it reopened.

**Pull request closed without merging**